css/src/components/popover.scss

```diff
@@ -0,0 +1,35 @@
+.popover {
+	display: inline-block;
+	position: relative;
+
+	.popover-summary {
+		color: $hyperlink;
+		list-style: none;
+		cursor: pointer;
+	}
+
+	.popover-content {
+		position: absolute;
+		min-width: 150px;
+		margin-top: $spacer-3;
+		padding: $spacer-6;
+		border: 1px solid $border;
+		border-radius: $border-radius;
```

Comment:
I think a few of these items can be extracted to variables with `!default` at the top of the file. 

bg color, box-shadow,  border (defaulting to or the only in high contract border),

Reply:
Updated

Reply:
Sorry, missed this in my previous comment, but I think border-radius as well

Reply:
You're right, updated!